examples/CustomEntitlementComputationSample/app/src/main/java/com/revenuecat/sample/main/MainScreen.kt

```diff
@@ -32,26 +36,59 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.compose.ui.window.Dialog
 import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
 import com.revenuecat.purchases.Package
+import com.revenuecat.sample.ui.theme.CustomEntitlementComputationTheme
 import com.revenuecat.sample.utils.findActivity
 
 @Composable
-fun MainScreen() {
+fun MainScreenNavigation() {
     val viewModel: MainViewModel = viewModel()
     val uiState = viewModel.uiState.collectAsState()
 
+    val navController = rememberNavController()
+    NavHost(navController, startDestination = "main") {
+        composable("main") { MainScreen(navController = navController) }
+        composable("customerInfoDetails/{date}") { backStackEntry ->
+            CustomerInfoDetailScreen(event = uiState.value.customerInfoList.find {
+                it.date.toString() == backStackEntry.arguments?.getString("date")
+            }!!)
+        }
+    }
+}
```

Comment:
nav controller here is useful for being able to navigate in and out of customer info details